### ISSUE TITLE
adding JFrog's public GPG key to OpenTofu registry

### ISSUE THREAD

**Issue by StephenWithPH** [jfrog/terraform-provider-artifactory#909]
**Is your feature request related to a problem? Please describe.**
When trying to use `terraform-provider-artifactory` via [OpenTofu](https://opentofu.org/), I receive the following error:

`Installed jfrog/artifactory v10.3.1. Signature validation was skipped due to the registry not containing GPG keys for this provider`

This arises because OpenTofu needs a member of JFrog's GitHub org to validate the public key used to sign provider releases.

**Describe the solution you'd like**
A member of JFrog's GitHub org needs to open a PR to `opentofu/registry` using this [PR form for signing keys](https://github.com/opentofu/registry/issues/new?assignees=&labels=provider-key%2Csubmission&projects=&template=provider_key.yml&title=Provider+Key%3A+JFrog).

**Describe alternatives you've considered**
N/A

**Additional context**
I believe submitting the GPG will cover all of JFrog's providers.

**Comment by alexhung** [comment]
@StephenWithPH Thanks for the head up. I've added this to our sprint.

**Comment by StephenWithPH** [comment]
> I believe submitting the GPG will cover all of JFrog's providers.

It occurred to me that my comment makes the assumption that JFrog uses the same signing key for all five of the providers. If that's not the case, you'll need to submit the signing key for each.

**Comment by alexhung** [comment]
@StephenWithPH See https://github.com/opentofu/registry/issues/314

**Comment by StephenWithPH** [comment]
Thank you for the quick turnaround! 🎉

**Comment by alexhung** [comment]
@StephenWithPH I submitted an expired public key the first time. You may see this error message `Error while installing jfrog/artifactory v10.1.4: authentication signature from unknown issuer` until my [new PR](https://github.com/opentofu/registry/pull/318) is synced at next cron job run (at top of the hour).

**Comment by StephenWithPH** [comment]
All good!

**Comment by alexhung** [comment]
@StephenWithPH All fixed!

<img width="643" alt="Screenshot 2024-03-15 at 11 30 15 AM" src="https://github.com/jfrog/terraform-provider-artifactory/assets/27263/d083f34d-75c6-46f8-8403-fa21ea8e097e">

**Comment by alexhung** [comment]
@StephenWithPH Turns out we were using a separate (older) signing key for Artifactory provider, whereas the other providers use the same (newer) key. Terraform registry allows for multiple signing keys but OpenTofu doesn't. Thus once I uploaded the old key to OpenTofu registry, the other 4 providers were no longer installable.

I've updated this provider to be signed with the newer key so this problem is solved. Right now we are [waiting for OpenTofu registry cache to be updated](https://github.com/opentofu/registry/issues/332#issuecomment-2004573189) before 10.3.3 is installable in OpenTofu.
